psiturk/psiturk_shell.py

```diff
@@ -988,10 +985,12 @@ def hit_create(self, numWorkers, reward, duration):
         if duration <= 0:
             print '*** duration must be greater than 0'
             return
-        
 
+        if not self._confirm_dialog('Create HIT? [y/n]: '):
```

Comment:
needs to respect `self.quiet` (see the two confirmation dialogs earlier in this method call) so that `psiturk -e` can still be reliably used